libp2p/utility.nim

```diff
@@ -0,0 +1,33 @@
+## Nim-LibP2P
+## Copyright (c) 2020 Status Research & Development GmbH
+## Licensed under either of
+##  * Apache License, version 2.0, ([LICENSE-APACHE](LICENSE-APACHE))
+##  * MIT license ([LICENSE-MIT](LICENSE-MIT))
+## at your option.
+## This file may not be copied, modified, or distributed except according to
+## those terms.
+
+import nimcrypto/utils
```

Comment:
`stew/byteutils` has a "better" `toHex` - in particular, we're trying to standardise on lowercase hex